src/cmd-build

```diff
@@ -429,6 +429,7 @@ else
         --label="coreos-assembler.image-input-checksum=${image_input_checksum}" \
         --label="org.opencontainers.image.source=${gitsrc}" \
         --label="org.opencontainers.image.revision=${config_gitrev}" \
+        --copymeta=fedora-coreos.stream \
```

Comment:
I guess this should be `--copymeta-opt` now?

Reply:
Yes, but we need an rpm-ostree release unfortunately right now.